README.md

```diff
@@ -0,0 +1,80 @@
+# OpenTelemetry Instrumentation: Spring and SpringBoot
+
+This package streamlines the manual instrumentation process in OpenTelemetry for Spring and SpringBoot. It will enable you to add traces to requests, and database calls with minimal changes to application code. This package will not fully automate your OpenTelemetry instrumentation, instead, it will provide you with better tools to instrument your own code.
```

Comment:
Note: SpringBoot is actually two words. I will suggest also adding explicit links to their official sites.

https://spring.io/projects/spring-framework
https://spring.io/projects/spring-boot